data/campaigns/Eastern_Invasion/_main.cfg

```diff
@@ -73,22 +153,73 @@
             name = "Neoriceisgood"
         [/entry]
     [/about]
-[/campaign]
+    [about]
+        title = _ "Campaign Redesign Playtesters"
+        [entry]
+            name = "Konrad2"
+        [/entry]
+        [entry]
+            name = "Lord-Knightmare"
+        [/entry]
+        [entry]
+            name = "Toranks"
+        [/entry]
+    [/about]
+    [about]
+        title = _ "Others"
+        [entry]
+            name = "Eric S. Raymond (ESR)"
+        [/entry]
+        [entry]
+            name = "Loci"
+        [/entry]
+    [/about]
 
 #ifdef CAMPAIGN_EASTERN_INVASION
-[binary_path]
-    path=data/campaigns/Eastern_Invasion
-[/binary_path]
+    {GLOBAL__ANIMATIONS_RECRUIT}
+    {GLOBAL__UNIVERSAL_AMLAS}
+    {GLOBAL__ABILITY_FORTIFY}
+    {GLOBAL__ABILITY_TERROR}
+    {GLOBAL__TRAIT_DESERTER}
+    {GLOBAL__TRAIT_MERCENARY}
+    {GLOBAL__IMPLMENTATION_SHIELD_OF_THE_SENTINEL}
+    {GLOBAL__PLAGUE_STAFF_RECRUIT_VARIATIONS}
+    {GLOBAL__PLAGUE_STAFF_USE_UPKEEP}
 
-[lua]
-    code="wesnoth.require 'campaigns/Eastern_Invasion/lua/bandits.lua'"
-[/lua]
+    # initialize right-click menus for the various items in the game.
+    # right-click menus last all campaign, but it doesn't hurt to redefine them every scenario
+    # EDIT - actually this does hurt, because now we have many duplicate events. Doesn't cause any problems for the player, but is messy.
+    #     still leaving it for now because it helps with debugging
+    {GLOBAL__DROPPABLE_HOLY_AMULET {ID_HOLY_AMULET_1}}
+    {GLOBAL__DROPPABLE_HOLY_AMULET {ID_HOLY_AMULET_2}}
+    {GLOBAL__DROPPABLE_HOLY_AMULET {ID_HOLY_AMULET_3}}
+    {GLOBAL__DROPPABLE_CRYSTAL_QUIVER}
+    {GLOBAL__DROPPABLE_SHIELD_OF_THE_SENTINEL}
+    {GLOBAL__DROPPABLE_BANEBLADE}
+    {GLOBAL__DROPPABLE_INVISIBILITY}
+    {GLOBAL__DROPPABLE_PLAGUE_STAFF}
 
-{campaigns/Eastern_Invasion/utils}
-[units]
-    {campaigns/Eastern_Invasion/units}
-[/units]
-{campaigns/Eastern_Invasion/scenarios}
-#endif
+    {ENABLE_PARAGON} # dunefolk are a challenge to get, so they may as well be powerful
+    {DISABLE_GRAND_MARSHAL} # don't let S17/S18b's generals overshadow Gweddry
+    [modify_unit_type]
+        type,set_advances_to=Ogre,Great Ogre
```

Comment:
so wait did the PR to add the Great Ogres get merged or not? How would that affect things?

Reply:
ok so yeah I was thinking of #7325 and it looks like that hasn't been merged yet